### PR TITLE
Release notes for Foreman 3.2

### PR DESCRIPTION
**Pull request by upadhyeammit**

### CODE REVIEW

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Feel free to merge this. Not sure if you want to wait for this until RC1 is available or merge it now.

**Comment by upadhyeammit** [comment]
> Feel free to merge this. Not sure if you want to wait for this until RC1 is available or merge it now.

Merging this now, once we have the Centos8 Stream pipeline green for 3.2 will merge: https://github.com/theforeman/theforeman.org/pull/1958

**Pull request merged**
